cr8/insert_fake_data.py

```diff
@@ -38,13 +38,19 @@
 """
 
 
+class Column(NamedTuple):
+    name: str
+    type_name: str
+    max_len: Optional[int]
```

Comment:
Maybe you can already use Python3's [Data Classes](https://docs.python.org/3/library/dataclasses.html) here?
```python
from dataclasses import dataclass


@dataclass
class Column:
    name: str
    type_name: str
    max_len: Optional[int] = None
```

In that manner, you can also spare passing the optional third parameter:
```python
>>> Column("foo", "bar")
Column(name='foo', type_name='bar', max_len=None)
```

Reply:
I think namedtuples are perfectly suited for the job here and don't see the advantage of using a data class.  The third argument isn't really optional in the production code,  it's just that it's nullable.